dds/DCPS/QOS_XML_Handler/QOS_XML_Loader.cpp

```diff
@@ -86,72 +86,72 @@ namespace DCPS {
+  QOS_XML_Loader::init(const ACE_TCHAR* qos_profile)
   {
     if (!qos_profile)
       {
         if (DCPS_debug_level > 5)
           {
-            ACE_ERROR ((LM_ERROR,
-              ACE_TEXT ("QOS_XML_Loader::init - ")
-              ACE_TEXT ("Passed an empty qos_profile, returning.\n")));
+            ACE_ERROR((LM_ERROR,
+              ACE_TEXT("QOS_XML_Loader::init - ")
+              ACE_TEXT("Passed an empty qos_profile, returning.\n")));
           }
         return ::DDS::RETCODE_BAD_PARAMETER;
       }
 
-    char *filename = this->get_xml_file_name (qos_profile);
+    ACE_TCHAR *filename = this->get_xml_file_name(qos_profile);
 
     if (!filename)
       {
         if (DCPS_debug_level > 5)
           {
-            ACE_ERROR ((LM_ERROR,
-              ACE_TEXT ("QOS_XML_Loader::init - ")
-              ACE_TEXT ("Unable to extract a file name from <%C>, returning.\n"),
+            ACE_ERROR((LM_ERROR,
+              ACE_TEXT("QOS_XML_Loader::init - ")
+              ACE_TEXT("Unable to extract a file name from <%C>, returning.\n"),
               qos_profile));
           }
         return ::DDS::RETCODE_BAD_PARAMETER;
       }
 
-    this->xml_file_.add_search_path (
+    this->xml_file_.add_search_path(
       ACE_TEXT("DDS_ROOT"),
       ACE_TEXT("/docs/schema/"));
 
-    DDS::ReturnCode_t const retcode =  this->xml_file_.init (filename);
+    DDS::ReturnCode_t const retcode =  this->xml_file_.init(filename);
 
-    ACE_OS::free (filename);
+    ACE_OS::free(filename);
 
     return retcode;
   }
 
   DDS::ReturnCode_t
-  QOS_XML_Loader::get_datawriter_qos (
+  QOS_XML_Loader::get_datawriter_qos(
     ::DDS::DataWriterQos& dw_qos,
-    const char * qos_profile,
-    const char * topic_name)
+    const ACE_TCHAR * qos_profile,
+    const ACE_TCHAR * topic_name)
   {
     if (!qos_profile)
       {
         if (DCPS_debug_level > 9)
           {
-            ACE_DEBUG ((LM_DEBUG,
-              ACE_TEXT ("QOS_XML_Loader::get_datawriter_qos - ")
-              ACE_TEXT ("No QOS profile provided. Can't do anything, ")
-              ACE_TEXT ("returning\n")));
+            ACE_DEBUG((LM_DEBUG,
+              ACE_TEXT("QOS_XML_Loader::get_datawriter_qos - ")
+              ACE_TEXT("No QOS profile provided. Can't do anything, ")
+              ACE_TEXT("returning\n")));
           }
 
         return DDS::RETCODE_OK;
       }
 
-    char* profile_name = this->get_profile_name (qos_profile);
+    ACE_TCHAR* profile_name = this->get_profile_name(qos_profile);
 
     if (!profile_name)
       {
         if (DCPS_debug_level > 5)
           {
-            ACE_ERROR ((LM_ERROR,
-              ACE_TEXT ("QOS_XML_Loader::get_datawriter_qos - ")
-              ACE_TEXT ("Error parsing profile string <%C>, returning.\n"),
+            ACE_ERROR((LM_ERROR,
+              ACE_TEXT("QOS_XML_Loader::get_datawriter_qos - ")
+              ACE_TEXT("Error parsing profile string <%C>, returning.\n"),
```

Comment:
Loggin tchar schould use %s instead of %C

Reply:
Thanks, fixed a bunch of those.